backend/controllers/bulkAppointmentsUpload.js

```diff
@@ -11,7 +11,9 @@ const bulkAppointmentsUploadFactory = (csvParserService, offenderLoader, logErro
   const index = async (req, res) => {
     const { data } = req.session
 
-    if (!data) return renderError(req, res)
+    if (!data) {
```

Comment:
:astonished: what you doing here @JamieDraperUK ?

Reply:
Not my code tbh. Changes made are probably formatting